src/components v2/Forms/Molecules/FormControl/FormControl.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="photon-form-control">
+    <P2Label
+      v-if="label"
+      v-bind="{
+        label,
+        inlineText,
+        isRequired,
+        size,
+      }"
+    />
+    <P2InputMessage
+      v-if="extraMessage"
+      v-bind="{
+        message: extraMessage,
+        icon: messageIcon,
+        size,
+      }"
+    />
+    <div class="photon-form-control__input">
+      <slot :size="size" />
+    </div>
+    <P2ErrorMessage
+      v-bind="{
+        error,
+        size,
+      }"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import P2Label from '@/components v2/Forms/Atoms/Label';
+import P2ErrorMessage from '@/components v2/Forms/Atoms/ErrorMessage';
+import P2InputMessage from '@/components v2/Forms/Atoms/InputMessage';
+
+export default Vue.extend({
```

Comment:
In `InputMessage` you removed `Vue.extend` and the vue import. Should we be following a consistent pattern in Photon for the Vue object?

Reply:
Yeah, I'm thinking about this. If I leave it with `Vue.extend` I can't reference props from atom components. If we want to keep the `Vue.extend` I'll have to see if there is an easy way to reference these prop values.

I'll have a look into this but you are right we need to be consistent.